### PR TITLE
Windows install

### PR DESCRIPTION
**Pull request by CharlieBlain**
Added windows install script

### CODE REVIEW

File: tools/install-hpegl-provider-windows.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Invoke-WebRequest $hpegl_dl_url -Out $hpegl_zip
+
+Write-Host Extracting release files
+Expand-Archive $hpegl_zip -Force
+
+Get-ChildItem -Path $hpegl -Recurse -File | Move-Item -Destination $dest_dir
+
+Remove-Item $hpegl_zip -Recurse -Force -ErrorAction SilentlyContinue 
+Remove-Item $hpegl -Recurse -Force -ErrorAction SilentlyContinue 
+Write-Host Complete
```
Comment (J0HNB0Y) [inline tools/install-hpegl-provider-windows.ps1:40]: Add a new line at the end

File: README.md [hunk 2]
```diff
@@ -25,3 +25,18 @@ The above will install the latest available release. To install a specific relea
...
+Use the script `tools/install-hpegl-provider-windows.ps1` to automatically download and install hpegl provider:
+
+```shell
+. { iwr -useb https://raw.githubusercontent.com/HewlettPackard/terraform-provider-hpegl/main/tools/install-hpegl-provider-windows.ps1 } | iex ;
+```
+
+The above will install the latest available release. To install a specific release:
+
+```shell
+iex "& { $(irm https://raw.githubusercontent.com/HewlettPackard/terraform-provider-hpegl/main/tools/install-hpegl-provider-windows.ps1) } -VERSION v0.0.1 -Preview "
```
Comment (J0HNB0Y) [inline README.md:41]: Whats the significance of "Preview" at the end here?

Comment (CharlieBlain) [reply README.md]: Don't think that was necessary, removed it.

**Review by eamonnotoole — COMMENTED** [review]
@CharlieBlain You need to rebase, README.md has changed since you created the branch.

Also, I presume that you have tested this on Windows?  To what level?  I think that as long as a "terraform init" can be run on Windows with an example .tf file and picks-up the correct local version then we'll be okay.

Use the examples/caas/main.tf from hpe-hcss/terraform-provider-hpegl if you're looking for an example to use.

**Review by zachovich — APPROVED** [review]

**Pull request merged**